### PR TITLE
chore: add new archetype to template [skip ci]

### PR DESCRIPTION
**Pull request by ZheSun88**
## Description

Please list all relevant dependencies in this section and provide summary of the change, motivation and context.

Fixes # (issue)

## Type of change

- [ ] Bugfix
- [ ] Feature

## Checklist

- [ ] I have read the contribution guide: https://vaadin.com/docs/latest/guide/contributing/overview/
- [ ] I have added a description following the guideline.
- [ ] The issue is created in the corresponding repository and I have referenced it.
- [ ] I have added tests to ensure my change is effective and works as intended.
- [ ] New and existing tests are passing locally with my change.
- [ ] I have performed self-review and corrected misspellings.

#### Additional for `Feature` type of change

- [ ] Enhancement / new feature was discussed in a corresponding GitHub issue and Acceptance Criteria were created.

### CODE REVIEW

File: scripts/generator/templates/template-release-notes.md [hunk 1]
```diff
@@ -103,22 +116,10 @@ Development is supported with the following operating systems:
 
 ## Desktop Browsers
 - Evergreen versions of the following browsers on :
```
Comment (manolo) [inline scripts/generator/templates/template-release-notes.md:118]: Remove `on`

File: scripts/generator/templates/template-release-notes.md [hunk 2]
```diff
@@ -103,22 +116,10 @@ Development is supported with the following operating systems:
...
-      - GTK+ 3.4 or higher
-      - GLib 2.22 or higher
-      - Pango 1.14 or higher
-      - X.Org 1.0 or higher (1.7 or higher is recommended)
-      - libstdc++ 4.6.1 or higher
-    - Latest Firefox ESR is supported (starting from Firefox ESR 68)
-  - Safari on macOS 10.15 or later (starting from Safari 13, available also for macOS 10.14.5 and 10.13.6)
-  - Edge on Windows 10 or later. Note: Vaadin 15 supports only Edge Chromium and does not provide support for EdgeHTML. 
+- Chrome on the operating systems [supported](https://support.google.com/chrome/a/answer/7100626?hl=en) by the product.
+- Firefox on the [supported](https://www.mozilla.org/en-US/firefox/91.0.1/system-requirements/) platforms
```
Comment (manolo) [inline scripts/generator/templates/template-release-notes.md:120]: replace 91.0.1 with 92.0

File: scripts/generator/templates/template-release-notes.md (hunk 1 above)
Comment (ZheSun88) [reply scripts/generator/templates/template-release-notes.md]: done

File: scripts/generator/templates/template-release-notes.md (hunk 2 above)
Comment (ZheSun88) [reply scripts/generator/templates/template-release-notes.md]: done

**Review by manolo — APPROVED** [review]

**Pull request merged**